resources/zooma/cttv_indications_3.txt

```diff
@@ -499,7 +499,8 @@ Genomics England Panel App		disease	Citrullinemia	http://www.orpha.net/ORDO/Orph
 Genomics England Panel App		disease	Cleft Lip with or without Cleft Palate	http://www.ebi.ac.uk/efo/EFO_0003959	Denise Carvalho-Silva, Ian Dunham, Gautier Koscielny, Naveen Aranganathan	2017-05-24 12:00:00
 Genomics England Panel App		disease	Cleft lip/Palate ectodermal dysplasia syndrome	http://www.orpha.net/ORDO/Orphanet_3253	Denise Carvalho-Silva, Ian Dunham, Gautier Koscielny, Naveen Aranganathan	2017-05-24 12:00:00
 Genomics England Panel App		disease	Cleft lip/palate-ectodermal dysplasia syndrome	http://www.orpha.net/ORDO/Orphanet_3253	Denise Carvalho-Silva, Ian Dunham, Gautier Koscielny, Naveen Aranganathan	2017-05-24 12:00:00
-Genomics England Panel App		disease	Cleft palate and mental retardation	http://www.orpha.net/ORDO/Orphanet_2014	Denise Carvalho-Silva, Ian Dunham, Gautier Koscielny, Naveen Aranganathan	2017-05-24 12:00:00
+Genomics England Panel App		disease	Cleft palate and mental retardation	http://purl.obolibrary.org/obo/MONDO_0016064	Irene Lopez	2021-01-19 12:00:00
+Genomics England Panel App		disease	Cleft palate and mental retardation	http://www.ebi.ac.uk/efo/EFO_0003847	Irene Lopez	2021-01-20 12:00:00

```

Comment:
Please change the order so that it still maps to `Cleft palate` while we sort [#1309](https://github.com/opentargets/platform/issues/1309) out